livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -668,4 +672,366 @@ void Lddc::PrepareExit(void) {
   }
 }
 
+void Lddc::initializeDiagnostics()
+{
+  using std::chrono_literals::operator""s;
+
+  bool check_pps_signal =
+    cur_node_->declare_parameter("check_pps_signal", false);
+
+  updater_.add("livox_tempereature", this, &Lddc::checkTemperature);
```

Comment:
Please change `tempereature` to `temperature`? (This is an original typo) :pray: